base/applications/mspaint/registry.cpp

```diff
@@ -223,7 +223,7 @@ void RegistrySettings::Store()
     CRegKey bar2;
     if (bar2.Create(paint, _T("General-Bar2")) == ERROR_SUCCESS)
     {
-        bar2.SetDWORDValue(_T("Bar#1"), Bar1Flags);
+        bar2.SetDWORDValue(_T("BarID"), Bar2ID);
```

Comment:
So here you changed the name, is it because this new name is the one compatible with Win2k3 mspaint?

Reply:
Sometimes not working.